scala-package/examples/src/main/scala/org/apache/mxnetexamples/util/DownloadUtil.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.mxnetexamples.util
+
+import java.io.File
+import java.net.URL
+
+import org.apache.commons.io.FileUtils
+
+object DownloadUtil {
```

Comment:
I think we don't need to create Separate classes for Utilities, we can put it under one class `Util`.  WDYT?

Reply:
I only see it being used in the example sections. The `Util` includes tools more for data processing

Reply:
if follow this convention we'll have to create a class for each Utility method, for example read image from file, reshape image(for the purpose of illustration), that would just create a bunch of classes within the Util package. Can we move this when the Util class becomes unmanageable?